src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/component/TestFixtures.kt

```diff
@@ -71,7 +71,11 @@ fun recallWithPopulatedFields(
   reasonsForRecallOtherDetail = randomString(),
   agreeWithRecall = AgreeWithRecall.values().random(),
   agreeWithRecallDetail = randomString(),
-  currentPrison = randomString()
+  currentPrison = randomString(),
+  additionalLicenceConditions = true,
+  additionalLicenceConditionsDetail = "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
+  differentNomsNumber = true,
+  differentNomsNumberDetail = "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua."
```

Comment:
A good example for me of the value of random data: we're using the same string to test this and `additionalLicenceConditionsDetail ` - with the implication of course that the code might confuse them but the test could still pass.